repository/cassandra.go

```diff
@@ -0,0 +1,89 @@
+package repository
+
+import (
+	"errors"
+
+	"github.com/gocql/gocql"
+	"go.uber.org/ratelimit"
+)
+
+type (
+	cassandraRepo struct {
+		session      *gocql.Session
+		shardDivisor int
```

Comment:
I think for this workshop, we don't need to use shard divisor @shivanivarshney3 , wdyt?

Reply:
oh actually Pranjal Saxena mentioned to me same, I asked him so we can teach them about how we strategise db for partitioning data in our storage